config/services.yaml

```diff
@@ -14,26 +14,21 @@ services:
         public: false
         bind:
           $elasticConfig: '%elastic%'
-
-          $queryBus: '@tactician.commandbus.query'
-          $commandBus: '@tactician.commandbus.command'
```

Comment:
keeping this you can avoid this https://github.com/jorge07/symfony-4-es-cqrs-boilerplate/pull/132/files#diff-55b5b167120840a9198b5779c9b1de66R42-R48 just by using the var name

Reply:
This is not necessary, we want use message buses only by `QueryBus` and `CommandBus` classes.

So new declaration forces for us to use this services.

Reply:
Yep. You're right